e2e/tests/assorted/04-setting.spec.ts

```diff
@@ -61,6 +61,10 @@ describe('Settings screen', () => {
 			await expect(element(by.id('settings-view-security-privacy'))).toExist();
 		});
 
+		it('should have media auto-download', async () => {
```

Comment:
Create a task as tech debt to properly test this

Reply:
https://rocketchat.atlassian.net/browse/NATIVE-355